packages/integration_test/lib/common.dart

```diff
@@ -121,3 +132,105 @@ class Failure {
     return Failure(failure['methodName'], failure['details']);
   }
 }
+
+/// Integration web tests can execute WebDriver actions such as screenshots.
+///
+/// These test will use [TestStatus] to notify `integration_test` of their
+/// state.
+enum TestStatus {
+  /// Test is waiting for executing WebDriver actions.
+  waitOnWebdriverCommand,
+
+  /// Test executed the previously requested action.
+  webdriverCommandComplete,
+}
+
+/// Types of different WebDriver actions that can be used in web integration
+/// tests.
+///
+/// These actions are either commands that WebDriver can execute or used
+/// for the communication between `integration_test` and the driver test.
+enum WebDriverActionTypes {
```

Comment:
In e2e, common is shared between driver side and the e2e side. They have to be public otherwise tests like these won't see them: https://github.com/flutter/plugins/blob/master/packages/integration_test/lib/integration_test_driver.dart#L72

Reply:
I added a comment to the top of the document (line 8-10)

Reply:
Do you think this entire file should be moved to lib/src to try and hide the symbols though? It's a bit weird that they will show up on the [API docs](https://pub.dev/documentation/integration_test/latest/common/common-library.html).

> I added a comment to the top of the document (line 8-10)

I can't find the comment, are you referring to this file?

Reply:
@jiahaog can you see it now?